frontend/src/components/Chat/ChatMessage.tsx

```diff
@@ -1,5 +1,30 @@
-function ChatMessage() {
-  return <li>ChatMessage</li>;
+import styled from 'styled-components';
+import colors from '../../styles/colors';
+import { MessageType } from '../../types/chat';
+
+interface ChatMessageProps {
+  chatMessage: MessageType;
+  isMine: boolean;
+}
+
+function ChatMessage({ chatMessage, isMine }: ChatMessageProps) {
+  return (
+    <StyledChatMessage isMine={isMine}>{chatMessage.message}</StyledChatMessage>
+  );
 }
 
 export default ChatMessage;
+
+const StyledChatMessage = styled.li<{ isMine: boolean }>`
+  padding: 0.5rem;
+  align-self: ${({ isMine }) => (isMine ? 'end' : 'start')};
+  color: ${({ isMine }) => (isMine ? colors.white : colors.black)};
+  background-color: ${({ isMine }) => (isMine ? colors.primary : colors.white)};
+  border: ${({ isMine }) => (isMine ? `none` : `1px solid ${colors.primary}`)};
+  border-radius: ${({ isMine }) =>
+    isMine
+      ? `8px 0px 8px 8px`
+      : `0px 8px 8px 8px;
+  `};
```

Comment:
```suggestion
${({ isMine }) => isMine ? css`
  color: colors.white;
` : css`
  coor: colors.black;
`;
```

이런 식으로 css 사용해서 한꺼번에 나타나는 방법도 있을 것 같아요~!

Reply:
아 이렇게 한번에 할 수도 있었네요... 숙련도 이슈입니다:<